### PR TITLE
Implement FunctorFilter for FluxK

### LINKED ISSUE #1501: Add FunctorFilter/MonadFilter instances for Rx and Reactor
https://twitter.com/sla_shalafi/status/1145988260004585472

### PR DESCRIPTION
**Pull request by lenguyenthanh**
Close #1501

### CODE REVIEW

**Comment by lenguyenthanh** [comment]
I realized that, Reactor's test package is wrong, it should be `arrow.fx` instead of `arrow.effect`. Should I fix it in this PR or another PR?

**Comment by raulraja** [comment]
It's fine to fix here, thanks!

**Comment by lenguyenthanh** [comment]
Fixed the wrong package.

File: modules/fx/arrow-fx-reactor/src/main/kotlin/arrow/fx/reactor/extensions/fluxk.kt [hunk 1]
```diff
@@ -198,3 +200,12 @@ interface FluxKTimer : Timer<ForFluxK> {
     FluxK(Mono.delay(java.time.Duration.ofNanos(duration.nanoseconds))
       .map { Unit }.toFlux())
 }
+
+@extension
+interface FuxKFunctorFilter : FunctorFilter<ForFluxK> {
```
Comment (pakoito) [inline modules/fx/arrow-fx-reactor/src/main/kotlin/arrow/fx/reactor/extensions/fluxk.kt]: Typo: `FluxKFunctorFilter`

Comment (lenguyenthanh) [reply modules/fx/arrow-fx-reactor/src/main/kotlin/arrow/fx/reactor/extensions/fluxk.kt]: Fixed!

**Comment by lenguyenthanh** [comment]
Hello, do I need to resolve conflicts in this PR?

**Comment by pakoito** [comment]
Oh, yes hahaha

**Comment by lenguyenthanh** [comment]
I merged master into this PR. CI on Bitrise seems to have some issue. It shows 500 internal error, so I don't know why it failed :(

**Comment by pakoito** [comment]
_ahem_

tap tap

**Comment by pakoito** [comment]
magic

**Comment by pakoito** [comment]
It's merged now, thank you for the instances.

**Pull request merged**